packages/drivers/driver-base/src/driverUtils.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Extract and return the w3c data.
+ * @param url - request url for which w3c data needs to be reported.
+ * @param initiatorType - type of the network call
+ * @param spReqDuration - Time spent on server reported by server in some servers.
+ */
+export function getW3CData(url: string, initiatorType: string, spReqDuration: number = 0) {
```

Comment:
I'd rather remove spReqDuration from here, and with it - networkTime.
It's trivial to calculate in a query - we have all the data. And name is very misleading (I'd rather rename fetchStartToResponseEndTime to be networkTime, though it's almost impossible to do in one go as we will break queries).

We can proactively notify everyone not to use networkTime - people can do it ahead of your changes.